src/template-renderer.js

```diff
@@ -5,6 +5,7 @@ const requireTemplate = require('./utils/require-template');
 
 module.exports = function() {
   return function(template, model, options, callback) {
```

Comment:
Just a note as I'm nothing this now, in the future we could simplify the signature of those API as `(options, callback) => {}`

Reply:
agree. We can do it in the future I think